tests/wandb_artifacts_test.py

```diff
@@ -864,6 +864,12 @@ def test_interface_commit_hash(runner):
         artifact.commit_hash()
 
 
+def test_artifact_incremental(runner):
+    artifact = wandb.Artifact(type="dataset", name="incremental-arty", incremental=True)
+    manifest = artifact.manifest.to_manifest_json()
+    assert manifest["type"] == "INCREMENTAL"
```

Comment:
This doesn't actually work, because the property I need to assert on is in the GQL request -- not the manifest file. I've experimented with putting lines in the mock server's graphql endpoint that log the variables to the context, but it doesn't seem like those lines are even getting run. Any tips?